cli_tools_e2e_test/common/utils/utils.go

```diff
@@ -35,9 +35,10 @@ type CLITestType string
 
 // List all test types here
 const (
-	Wrapper                   CLITestType = "1 wrapper"
-	GcloudProdWrapperLatest   CLITestType = "2 gcloud-prod wrapper-latest"
-	GcloudLatestWrapperLatest CLITestType = "3 gcloud-latest wrapper-latest"
+	Wrapper                       CLITestType = "1 wrapper"
+	GcloudBetaProdWrapperLatest   CLITestType = "2 gcloud-beta-prod wrapper-latest"
```

Comment:
The format like gcloud-beta-prod is a bit hard to understand. Maybe find a a more clear way to describe it,

Reply:
The format is <command>-<release track>-<code branch> which more precisely indicate what is being tested than only <command>-<code branch> as we had before. 

Do you have a suggestion how to make it easier to understand?

Reply:
gcloud(beta)-prod lokks more clear to me

Reply:
Changed